_includes/_head.htm

```diff
@@ -8,13 +8,12 @@
 	<meta name="author" content="https://github.com/grctest/Gridcoin-Site/graphs/contributors">
 	<meta name="description" content="{{ page.description | escape }}">
 
-	<!--     Fonts and icons     -->
-	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,700,200" rel="stylesheet" />
+	<!--     Fonts and icons     
+	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,700,200" rel="stylesheet" />-->

```

Comment:
There is no reason given for this commenting out.  Is this part of the PR?

Reply:
After updating Bootstrap the font changed and messed with the layout of the site. Commenting this out fixed it to a degree.

I should really fix the Font once and for all but we have 3 things messing with it. Bootstrap does stuff to the font, NowUI messes with what bootstrap does and then we manually define a font as well. 

I'd sooner just use bootstraps default font or one of the fonts it has with it.

I don't know much about setting up fonts for websites. I should look into it.

Reply:
Hang on, I got a fix for this, I can reinstate the font, Just have to fix a few oddly sized bits of text on the index page.

Reply:
This is with the font,
![screencapture-127-0-0-1-4000-2019-05-20-15_25_59](https://user-images.githubusercontent.com/772631/58029545-b94e7580-7b14-11e9-9c89-975c28bd1614.png)

This is without the font. It looks better without but ideally, I'd like to remove Now-ui so bootstraps responsive font stuff stops getting overridden.

![Uploading screencapture-127-0-0-1-4000-index-htm-2019-05-20-15_33_31.png…]()

Reply:
Recommend a new PR to revisit the font issue separately.  Then I could just give the current PR the okay.  I think the font discussion should be a separate one.